Doc/c-api/bytes.rst

```diff
@@ -72,28 +72,28 @@ called with a non-bytes parameter.
    | :attr:`%c`        | int           | A single byte,                 |
    |                   |               | represented as a C int.        |
    +-------------------+---------------+--------------------------------+
-   | :attr:`%d`        | int           | Exactly equivalent to          |
+   | :attr:`%d`        | int           | Nearly equivalent to           |
    |                   |               | ``printf("%d")``.              |
```

Comment:
Please add a reference to the note, like (1).

I would prefer to remove Nearly/Exactly. I don't think that the subtle difference is major enough to justify a "Nearly".